code/modules/projectiles/guns/projectile/automatic/z8.dm

```diff
@@ -1,36 +1,34 @@
 /obj/item/weapon/gun/projectile/automatic/z8
-	name = "FS AR 5.56x45mm \"Z8 Bulldog\""
-	desc = "The Z8 Bulldog is an older model bullpup carbine, made by \"Frozen Star\". Uses 5.56mm rounds."
+	name = "FS CAR 5.56x45mm \"Z8 Bulldog\""
+	desc = "The Z8 Bulldog is an older model bullpup carbine, made by \"Frozen Star\". Has underbarrel grenade launcher which are compitable with most modern grenade types. Uses 5.56mm rounds."
```

Comment:
desc = "The Z8 Bulldog is an older bullpup carbine model, made by \"Frozen Star\". It includes an underbarrel grenade launcher which is compatible with most modern grenade types. Uses 5.56mm rounds."